Octokit.Tests/Clients/ReactionsClientTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using NSubstitute;
+using Octokit;
+using Octokit.Tests;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+public class ReactionsClientTests
+{
+    public class CommitComments
+    {
+        public class TheGetAllMethod
+        {
+            [Fact]
+            public async Task RequestsCorrectUrl()
+            {
+                var connection = Substitute.For<IApiConnection>();
+                var client = new ReactionsClient(connection);
```

Comment:
So you would then check each subclient in each method? Might make the tests a bit noisy.
Also, I would suggest changing the name of the class [here](https://github.com/octokit/octokit.net/pull/1335#discussion-diff-65077052R10)